src/MercadoPago/Net/MPRequest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace MercadoPago\Net;
+
+/** MPRequest class. */
+class MPRequest
+{
+    private $uri;
+
+    private $method;
+
+    private $headers;
+
+    private $payload;
+
+    private $connection_timeout;
+
+    /**
+     * MPRequest constructor.
+     * @param string $uri path to be requested.
+     * @param string $method method to be used.
+     * @param mixed $payload payload to be sent.
+     * @param mixed $headers headers to be sent.
+     * @param mixed $connection_timeout connection timeout to be sent.
+     */
+    public function __construct(string $uri, string $method, ?string $payload = null, ?array $headers = null, ?int $connection_timeout = null)
```

Comment:
@rhames07 chegou a explorar a opção de usar constructor property promotion do php8 ? Aqui tem um exemplo: https://www.tutorialspoint.com/constructor-property-promotion-in-php-8 . Se fosse possível, diminuiria a quantidade de linhas da classe.